src/main/resources/hudson/views/BuildStatusFilter/config.jelly

```diff
@@ -3,9 +3,15 @@
 	xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form"
 	xmlns:i="jelly:fmt" xmlns:p="/lib/hudson/project">
   <f:entry title="${%Build Statuses}:">
-    <f:checkbox name="building" field="building"/> ${%Currently Building}
-    <f:checkbox name="neverBuilt" field="neverBuilt"/> ${%Never Built}
-    <f:checkbox name="inBuildQueue" field="inBuildQueue"/> ${%In Build Queue}
+    <div>
+      <f:checkbox title="${%Currently Building}" field="building"/>

```

Comment:
uses labels in stead of random text place

`<div>` to make it a block element so that checkboxes go line by line